v3/src/models/data/formula-mathjs-scope.ts

```diff
@@ -2,6 +2,8 @@ import { AGGREGATE_SYMBOL_SUFFIX, GLOBAL_VALUE, LOCAL_ATTR } from "./formula-typ
 import type { IGlobalValueManager } from "../global/global-value-manager"
 import type { IDataSet } from "./data-set"
 
+const CACHE_ENABLED = true
```

Comment:
I agree, but I'll leave it for a separate PR probably. This flag is set to disabled in the last open PR, so I'm not gonna change it now to avoid merge conflicts.